packages/table-react/src/ExpandableTableRow.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { FC, useState } from "react";
+import cx from "classnames";
+import { useAnimatedHeight, useId } from "@fremtind/jkl-react-hooks";
+import { ExpandableTableRowController } from "./ExpandableTableRowController";
+import { TableRowProps, TableRow } from "./TableRow";
+
+export interface ExpandableTableRowProps extends TableRowProps {
+    expandedChildren: React.ReactNode;
+    /**
+     * Setter bredden på raden som blir åpnet
+     * @default 100
+     */
+    colSpan?: number;
+}
+
+export const ExpandableTableRow: FC<ExpandableTableRowProps> = ({
+    className,
+    clickable,
+    children,
+    expandedChildren,
+    colSpan = 100,
+    ...rest
+}) => {
+    const [isOpen, setIsOpen] = useState(false);
+    const [animationRef] = useAnimatedHeight<HTMLDivElement>(isOpen);
```

Comment:
Denne har noen callbacks som kanskje kunne vært fornuftige å eksponere her? Hvis man f.eks. vil resette noe state når innholdet er lukket/før det åpnes e.l. Da må man i så fall tillate en render-funksjon for `expandedChildren` som får inn disse.